nueljoe/healthcare
# feature(user-profile):implement profile endpoints

#### What does this PR do?
Implement profile and profile-related endpoints

#### Description of Task to be completed?
- implement the `GET /me` endpoint to fetch the personal details of the current user

- implement the `PATCH /me` endpoint to update the personal details of the current user

- implement the `PATCH /me/avatar` endpoint to update avatar of the current user

- implement the `GET /me/enrollments` endpoint to fetch a paginated list of courses in which the current user is enrolled.

- implement the `GET /me/payments` endpoint to fetch the payment history of the current user

- implement the `GET /me/courses` endpoint to fetch a paginated list of courses created by the current user

#### How should this be manually tested?
- clone the repository
- cd into the project directory and run `cd api` to CD into the `api` directory
- see the README.md file in the `api` directory for the preliminary steps
- ensure that a mysql database in installed and running on your device
- see the postman documentation on course enrollment and `Profile`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A
#### Questions:
